src/main/java/com/tourism/psk/controller/TripController.java

```diff
@@ -21,22 +22,40 @@ public TripController(TripService tripService) {
     }
 
 
-    @RequestMapping(value = "groupTrip", method = RequestMethod.POST, produces = MediaType.APPLICATION_JSON_VALUE)
+    @RequestMapping(value = "/groupTrip", method = RequestMethod.POST,
```

Comment:
The endpoints should be GET /group-trip, PUT /group-trip GET /group-trip/{id} and PATCH (don't know why that method specifically, might as well be PUT) /group-trip/{id}. See [updated swagger](https://github.com/tautvydass/vapersoft-gecko/blob/master/documentation/specification.yaml) for more details 
Since the endpoints made before follow this scheme, we might as well be consistent.

Reply:
Also each endpoint should call SessionService.authenticate before doing anything else.

Reply:
use - in endpoints is not best practise, these ones should suffice